src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp

```diff
@@ -208,6 +240,69 @@ void CobConverter::UpdateState(uint64_t CurrentSimNanos)
     this->opnavCOMOutMsg.write(&comMsgBuffer, this->moduleID, CurrentSimNanos);
 }
 
+/*! Compute the total COB covariance matrix in pixel units (given unit vector covariances)
+    @param Eigen::Matrix3d covarNav_N
+    @param Eigen::Matrix3d covarAtt_B
+    @param Eigen::Matrix3d covarCob_C
+    @param Eigen::Matrix3d dcm_CN
+    @param Eigen::Matrix3d dcm_CB
+    @param Eigen::Matrix3d cameraCalibrationMatrix
+    @return Eigen::Matrix3d covarImage
+    */
+Eigen::Matrix3d CobConverter::computeTotalCobCovariance(Eigen::Matrix3d covarNav_N,
```

Comment:
Let's change all of these to references rather than values.

Reply:
Done. Also changed it for the cobOutlierDetection method.